components/automate-ui/src/app/services/http/http-client-auth.interceptor.spec.ts

```diff
@@ -36,7 +36,7 @@ describe('HttpClientAuthInterceptor', () => {
 
     it('when a 401 response is intercepted logs out the session', done => {
       spyOn(chefSession, 'logout');
-      httpClient.get('/endpoint').subscribe(null, done);
+      httpClient.get('/endpoint').subscribe({ error: done });
```

Comment:
Here's a good example showing why the "observer object" is better than separate arguments ("next, error, complete" in that order). This test wants to ignore the `next` callback, so it gave a null. It then wants to provide an `error` callback, because the http call is expected to fail, and that is the jasmine `done`. So in the revised code, we just provide the `error` callback--no guesswork involved.